.rubocop.yml

```diff
@@ -46,23 +48,42 @@ Style/DotPosition:
 Style/EmptyLineBetweenDefs:
   AllowAdjacentOneLineDefs: true
 
+# Maintain compatibility with older Rubies
```

Comment:
But the freeze pragma will just be ignored by older rubies, won't it?
I thought about adding the pragma everywhere as well ...